indico/modules/categories/templates/management/delete_categories.html

```diff
@@ -0,0 +1,15 @@
+{% from 'confirmation_dialog.html' import confirmation_dialog %}
+
+{% call confirmation_dialog('danger', ok_text=ngettext("Delete category", "Delete categories", categories|length),
+                            extra_confirmation=false, id_field_name='category_id', ids=category_ids) %}
+    {% trans count=categories|length %}
+        You are about to delete the following category:
+    {% pluralize %}
+        You are about to delete the following categories:
```

Comment:
same here. it results in awful i18n dictionaries if you don't (there'd be surrounding whitespace in the extracted strings)